tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/ArchetypesTest.java

```diff
@@ -69,12 +94,21 @@
 
 	@BeforeClass
 	public static void setup() {
-		String mvnConfigFileName = new File(MAVEN_SETTINGS_PATH).getAbsolutePath();
 		WorkbenchPreferenceDialog preferenceDialog = new WorkbenchPreferenceDialog();
 		preferenceDialog.open();
+
+		JREsPreferencePage jrePreferencePage = new JREsPreferencePage(preferenceDialog);
+		preferenceDialog.select(jrePreferencePage);
+		List<TableItem> tis = new DefaultTable(0).getItems();
+		for (TableItem ti : tis) {
+			if (ti.getText(0).startsWith("Java SE 8") || ti.getText(0).startsWith("openjdk-1.8")) {
+				ti.setChecked(true);
+				jrePreferencePage.apply();
+			}
+		}
+
 		MavenSettingsPreferencePage prefPage = new MavenSettingsPreferencePage(preferenceDialog);
```

Comment:
I suppose you do not need this stuff, because you have removed maven config file stuff.

Reply:
Removed.